src/Uno.UI/UI/Xaml/Controls/ListViewBase/NativeListViewBaseAdapter.Android.cs

```diff
@@ -143,9 +148,14 @@ private ContentControl GenerateContainer(int viewType)
 				return XamlParent?.GetGroupHeaderContainer(null);
 			}
 
-			return XamlParent?.GetContainerForIndex(-1) as ContentControl;
+			//return XamlParent?.GetContainerForIndex(-1) as ContentControl;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5961910)